src/Console/StartWebSocketServer.php

```diff
@@ -104,6 +109,19 @@ protected function configureConnectionLogger()
         return $this;
     }
 
+    public function configureRestartTimer()
+    {
+        $this->lastRestart = $this->getLastRestart();
+
+        $this->loop->addPeriodicTimer(10, function () {
+            if ($this->getLastRestart() != $this->lastRestart) {
```

Comment:
Can you make the check a strict check (`!==`) ?

Reply:
Done